demo/demo-tekton.sh

```diff
@@ -221,17 +239,33 @@ printf "Templates: "
 oc delete --all template.template.openshift.io -n "${app_namespace}" 2> /dev/null || echo "...done"
 printf "RBAC Authorization: "
 oc delete "clusterrolebinding.rbac.authorization.k8s.io/pipeline-role-binding-${app_namespace}" 2> /dev/null || echo "...done"
-pv="$(oc get pv | grep "${app_name}" | cut -d " " -f 1)"
-pvc="$(oc get pvc -n "${app_name}" | cut -d " " -f 1 | tail -n 1)"
-printf "Remove PV's and PVC's associated w/ the tekton demo: %s and %s\n" "$pv" "$pvc" 
-oc delete pv "$pv" --grace-period=0 --wait=false || true
-oc delete pvc "$pvc" -n "${app_namespace}" --grace-period=0 --wait=false || true
-oc patch pvc "$pvc" -p '{"metadata":{"finalizers":null}}' -n "${app_namespace}" || true
+if oc get pv | grep "${app_name}"; then
+    pv="$(oc get pv | grep "${app_name}" | cut -d " " -f 1)"
+    printf "Remove PV's associated w/ the tekton demo: %s\n" "$pv"
+    oc delete pv "$pv" --grace-period=0 --wait=false || true
+else
+    echo "No PV's to remove"
+fi
+app_pvcs=$(oc get pvc -n "${app_name}" 2>&1)
+if [[ $app_pvcs == "No resources found in ${app_name} namespace." ]]; then
+    echo "No PVC's to remove"
+else
+    pvc="$(oc get pvc -n "${app_name}" | cut -d " " -f 1 | tail -n 1)"
+    printf "Remove PVC's associated w/ the tekton demo: %s\n" "$pvc"
+    oc delete pvc "$pvc" -n "${app_namespace}" --grace-period=0 --wait=false || true
+    oc patch pvc "$pvc" -p '{"metadata":{"finalizers":null}}' -n "${app_namespace}" || true
+fi
+# oc delete -n "${app_namespace}" -f /tmp/05-build-and-deploy.yaml.out
+# oc process -f "$tekton_setup_dir/04-service-account_template.yaml" -p PROJECT_NAMESPACE="${app_namespace}" -n default | oc delete -f -
+# oc delete -f "$tekton_setup_dir/03-rbac-pipeline-role.yaml"
+# oc delete -f "$tekton_setup_dir/02-tekton-operator.yaml" # not enough :(
+# exit 0
 
 echo "Setting up resources:"
 
 echo "2. Installing tekton operator"
 oc apply -f "$tekton_setup_dir/02-tekton-operator.yaml"
+# I think we need to wait here
```

Comment:
For what?

Reply:
I believe tekton operator does build immediately. The first time I run with the update 02 file, it still broke, then it passed in the second time. I suspect because it was still building